README.md

```diff
@@ -11,32 +11,85 @@ This repo defines the web interface for various drunc tools. Including:
 - The Processes Manager
 - The Controller
 
-## Running the App
+## Docker Setup
 
-To run with a demo version of the drunc process manager, run it with docker compose:
+This repository provides a Docker Compose configuration providing the requisite services
+suitable for development of the interfaces using realistic test data. Working with the
+full docker stack requires building a very larger docker base image and has heavy
+resource requirements when running. For this reason a "lite" stack is also provided that
+can be used for development of the Process Manager UI.
+
+The full and lite stack are defined using [Docker Compose profiles]. It is recommended
+to set the `COMPOSE_PROFILES` environment variable to select which stack you are working
+with. Use `COMPOSE_PROFILES=drunc` for the full stack or `COMPOSE_PROFILES=drunc-lite`
+for the lite stack.
+
+You can also use the `--profile` flag when invoking `docker compose` however the
+instructions in this file assume you are using the environment variable.
+
+[Docker Compose profiles]: https://docs.docker.com/compose/how-tos/profiles/
+
+## Process Manager UI
+
+You can use either the full or lite Docker Compose profiles when working with the
+Process Manager UI.
+
+### Getting Started with the Process Manager UI
+
+Check you have a value set for `COMPOSE_PROFILES` (see above). Start the docker stack
+with:
 
 ```bash
 docker compose up
 ```
 
-It can take a few moment for the services to boot but the application should then be
-available in the browser at <http://localhost:8000>.  Authentication is required to work
-with the application so you need to create a user account to work with:
+It can take a few moments for the services to boot but the application should then be
+available in the browser at <http://localhost:8000>.
+
+__Note that if you have the Process Manager UI already open in a browser you may see
+backtraces from unsuccessful requests generated by the UI. These should stop once drunc
+has finished starting up.__
```

Comment:
Are the backtraces full stack traces? Will users be able to see it in production....? If so, is that what we want?

Reply:
Ah, maybe I should be clearer here. It's only in the logs in the terminal.

Reply:
Ohh I see. That's not such a big deal